src/node_report.cc

```diff
@@ -1020,6 +1024,11 @@ static void PrintGCStatistics(std::ostream& out, Isolate* isolate) {
  ******************************************************************************/
 static void PrintResourceUsage(std::ostream& out) {
   char buf[64];
+  double cpu_abs;
+  double cpu_percentage;
+  long int diff_time = current_time - load_time;
```

Comment:
I think the time() function gives the time in seconds:
http://en.cppreference.com/w/cpp/chrono/c/time